web-24wi/assignments/AkinaSS/infra/week5/coffeemasters-authn-main/server.js

```diff
@@ -0,0 +1,42 @@
+import express from 'express'
+import { Low } from 'lowdb'
+import { JSONFile } from 'lowdb/node'
+import * as url from 'url';
```

Comment:
Does this work with .js as a file extension or do you need to have .cjs as a file extension.

Reply:
Oh this is the server.js from the frontendmaster course, it was default like that so I don't know but I'm pretty sure it should be working in a degree if I followed the class